crates/nu-command/src/filters/empty.rs

```diff
@@ -125,30 +126,15 @@ fn empty(
                 span: head,
             }
             .into_pipeline_data()),
-            PipelineData::Value(value, ..) => {
-                let answer = is_empty(value);
-
-                Ok(Value::Bool {
-                    val: answer,
-                    span: head,
-                }
-                .into_pipeline_data())
+            PipelineData::Value(value, ..) => Ok(Value::Bool {
+                val: value.is_empty(),
+                span: head,
             }
+            .into_pipeline_data()),
         }
     }
 }
 
-pub fn is_empty(value: Value) -> bool {
-    match value {
-        Value::List { vals, .. } => vals.is_empty(),
-        Value::String { val, .. } => val.is_empty(),
-        Value::Binary { val, .. } => val.is_empty(),
-        Value::Nothing { .. } => true,
-        Value::Record { cols, .. } => cols.is_empty(),
-        _ => false,
-    }
-}
-
```

Comment:
For this part - do we really want to move away from checking whether something is empty in a more fuzzy way? For example - this came up a few ways:

```
> [] | empty?
```
Now you can check that the list is empty. Similarly for empty records, etc.

Reply:
Not sure I follow. In any case, this seems duplicate code since `is_empty()` already [exists here](https://github.com/nushell/nushell/pull/5214/files/822e9e6aaf97e239aa188e52c74d5671195e01e3#diff-654f056f1c72a4868170c32aa44d6dd497a83cb1c8b919e24a7dc50e8f81e624R574-R582).

Thoughts?